BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/Internal/BrainCloudComms.cs

```diff
@@ -1945,6 +1979,28 @@ public void EnableComms(bool value)
         {
             _enabled = value;
         }
+        
+        private static bool IsJsonValid(string strInput)
+        {
+            if (string.IsNullOrWhiteSpace(strInput)) { return false;}
+            strInput = strInput.Trim();
+            if ((strInput.StartsWith("{") && strInput.EndsWith("}")) || //For object
+                (strInput.StartsWith("[") && strInput.EndsWith("]"))) //For array
+            {
+                try
+                {
+                    var obj = JsonReader.Deserialize<JsonResponseBundleV2>(strInput);
```

Comment:
Ok, so we deserialize the json twice. Very bad. Some people have _huge_ entities that actually cause game lag when desereliazing.
What I propose is rename this function to `DeserializeJson()`. Return the `obj`, or `null` if it failed. So you can still check for validity.

Reply:
Good call, will do !